featuretools/primitives/standard/transform/numeric/rate_of_change.py

```diff
@@ -0,0 +1,35 @@
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Double
+
+from featuretools.primitives.base import TransformPrimitive
+
+
+class RateOfChange(TransformPrimitive):
+    """Computes the rate of change of a value per second.
```

Comment:
I think it may be interesting in the future to have the unit of time be configurable

Reply:
I also thought about that, but I don't think having a configurable unit would add predictive value since changing the unit effectively just scales the output by a fixed value. Might be useful in other circumstances though. My preference is to keep this simple until we identify a specific need for the additional complexity.

Reply:
I agree. I think it would be more of a convenience for the user if they wanted to examine their feature matrix. For example, if the user had data sampled daily.